src/lib/platform/EiScreen.cpp

```diff
@@ -423,7 +423,10 @@ bool EiScreen::isPrimary() const
 
 void EiScreen::update_shape()
 {
-
+    w_ = 0;
```

Comment:
When the class is created w_ and h_ are set to `1` not 0 .

Reply:
Should be good now !